packages/react-components/src/theming/icons.tsx

```diff
@@ -147,5 +147,7 @@ export const DEFAULT_COMPONENT_ICONS = {
   /* @conditional-compile-remove(dialpad) */ /* @conditional-compile-remove(PSTN-calls) */
   BackSpace: <Backspace20Regular />,
   /* @conditional-compile-remove(call-readiness) */
-  iconBackdrop: <Circle24Filled />
+  iconBackdrop: <Circle24Filled />,
+  /* @conditional-compile-remove(call-readiness) */
+  sparkle: <Sparkle20Filled />
```

Comment:
When you do add this (in a separate PR), it should be named to signify where it isused, not what icon it is.

The names we define should be domain specific. This way Contoso knows what overriding that icon does, and it also avoids conflicts with the default icon names in Fluent.